be/src/runtime/tablets_channel.cpp

```diff
@@ -476,10 +476,9 @@ Status TabletsChannel::add_batch(const TabletWriterAddRequest& request,
         if (!st.ok()) {
             auto err_msg = strings::Substitute(
                     "tablet writer write failed, tablet_id=$0, txn_id=$1, err=$2"
-                    ", errcode=$3, msg:$4",
-                    tablet_to_rowidxs_it.first, _txn_id, st.code(), st.precise_code(),
-                    st.get_error_msg());
-            LOG(WARNING) << err_msg;
+                    ", precise_code=$3",
+                    tablet_to_rowidxs_it.first, _txn_id, st.code(), st.precise_code());
+            LOG(WARNING) << err_msg << " bt:" << st.get_error_msg();
```

Comment:
Please do not use error code or precise code as error message. Better to use exception stack since we may remove  error code mechanism in the future. We will use exception instead.

Reply:
I think put a full stack in error message for user is confusing? stack is used for developer not user

Reply:
both of you make sense ... lMHO let's just use error code here and leave it as future work until whatever alternative mechanism takes place?